pyproject.toml

```diff
@@ -119,63 +119,7 @@ skip_glob = ["typings/*", "docs/*"]
 
 [tool.pyright]
 pythonVersion = "3.10"
-include = ["bentoml", "typings"]
+include = ["bentoml"]
 exclude = ['bentoml/_version.py','bentoml/__main__.py']
-strictListInference = true
-strictDictionaryInference = true
-strictSetInference = true
```

Comment:
why do we remove these

Reply:
I wanted to go back to full strict mode and just `# type: ignore` all the unknownmembertype and missingstubs issues.

Reply:
Is full strict mode stricter or looser than the current?

Reply:
stricter

Reply:
I prefer not to do too much style changes till a stable code base.